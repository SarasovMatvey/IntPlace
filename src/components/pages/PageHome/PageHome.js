import { Container } from "semantic-ui-react";
import PlacesList from "../../organisms/PlacesList";
import BaseTemplate from "../../templates/BaseTemplate/BaseTemplate";

function PageHome() {
  return (
    <BaseTemplate>
      <div className="page-home">
        <Container>
          <PlacesList />
        </Container>
      </div>
    </BaseTemplate>
  );
}

export default PageHome;
