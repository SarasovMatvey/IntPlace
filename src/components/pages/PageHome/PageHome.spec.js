import PageHome from "./PageHome";

const setUp = props => shallow(<PageHome {...props} />);

describe("PageHome", () => {
  let component;

  beforeEach(() => {
    component = setUp();
  });

  it("should renders properly", () => {
    expect(component).toMatchSnapshot();
  });
});
