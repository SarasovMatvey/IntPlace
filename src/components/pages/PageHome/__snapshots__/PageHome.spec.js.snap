// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHome should renders properly 1`] = `
<BaseTemplate>
  <div
    className="page-home"
  >
    <Container>
      <PlacesFinder />
    </Container>
  </div>
</BaseTemplate>
`;
